CONTRIBUTING.md

```diff
@@ -1,14 +1,49 @@
 # Contributing to fortran-lang.github.io
 
 Fortran-lang.github.io is open-source and contributions are welcome!
-The Fortran-lang site uses the Ruby-based [Jekyll static site generator](https://jekyllrb.com/).
-To contribute you will therefore need to install Jekyll on your development computer.
-See [README.md](README.md) for how to setup Jekyll and build the site.
 
 * See [PACKAGES](./PACKAGES.md) for how to add an entry to the [Package index](https://fortran-lang.org/packages)
 
 * See [MINIBOOKS](./MINIBOOKS.md) for how to write and structure a mini-book tutorial for the [Learn](https://fortran-lang.org/learn) section
 
+
+## Introduction
+
+__How is the site written?__
+
+The content of the website is primarily written in a combination of _markdown_, HTML and YAML (for data).
+This source is compiled to produce pure HTML which is what you see on the final website.
+
+The website is _static_ which means that once built, the content on the site is the same for all users;
+this is in contrast to many websites which are _dynamic_, meaning they can serve different content 
+depending on the user and the inputs supplied by the user.
+
+Structural components of the website are written in the Jekyll [Liquid](https://github.com/Shopify/liquid/wiki) templating language for static features, and javascript for dynamic features.
+
+
+__Do I need to know HTML to contribute?__
+
+The majority of the site content is written in [markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) which is a simplified markup language for formatting text - don't worry if you haven't used it before, it's very easy to pick up!
+
+
+__How is the site built?__
+
+The Fortran-lang site uses the Ruby-based [Jekyll static site generator](https://jekyllrb.com/)
+to compile the markdown and HTML files.
+To contribute you will therefore need to install Jekyll on your development computer, so that you
+can build the website on your computer and preview your changes.
+(The compiled website can be found in the `_site` folder.) 
+See [README.md](README.md) for how to setup Jekyll and build the site.
```

Comment:
I worry that this may be discouraging to some less-technical users. It's actually quite possible to contribute to the website by adding or editing Markdown files and using the preview feature. Would it be reasonable to state here that it's _recommended_ to be able to run Jekyll locally, but not necessary? (and then we point the reader to the build preview feature below.

Reply:
Yes good point - I will update.